src/Features/Lsif/GeneratorTest/HoverTests.vb

```diff
@@ -42,28 +42,46 @@ class C
             Dim expectedHoverContents As String
             Select Case code
                 Case "class [|C|] { string s; }"
-                    expectedHoverContents = "class C"
+                    expectedHoverContents = "```csharp
+class C
+```
+  "
```

Comment:
What's with this magic two spaces here?

Reply:
Holy crap that's an, um, _interesting_ markdown behavior. I don't know why we're doing a single-spaced newline at the end of the contents though if there's nothing after it, but that's probably just a common issue with the hover thing so nothing unique here.

Reply:
indeed, see https://daringfireball.net/projects/markdown/syntax#p .  And you're right it probably shouldn't do that when the linebreak is the last 'tag'.